acl2_kernel/kernel.py

```diff
@@ -38,7 +38,18 @@ def __init__(self, **kwargs):
     def _start_acl2(self):
         sig = signal.signal(signal.SIGINT, signal.SIG_DFL)
         try:
-            self.acl2wrapper = replwrap.REPLWrapper(os.environ['ACL2'], 'ACL2 !>', None)
+            prompt_change_cmd = '''
+(progn
+  (set-state-ok t)
+  (defun jupyter-prompt (channel state)
+    (declare (xargs :mode :program))
+    (fmt1 "JPY-ACL2>" '() 0 channel state nil))
+  (set-state-ok nil))
+(set-ld-prompt 'jupyter-prompt state)
+(reset-prehistory)'''
```

Comment:
Could you add some indent like;

```
            prompt_change_cmd = '''
		    (progn
		      (set-state-ok t)
		      (defun jupyter-prompt (channel state)
```

Reply:
Sure 7259106e22c909db7ae9d7a2a3a9092bd864d992

Reply:
Thanks!